doc/run/workflow_mode.md

```diff
@@ -0,0 +1,91 @@
+# Running SQLFlow as Workflow Mode
+
+This tutorial introduces how to setup the SQLFlow with workflow mode, you can
+
+1. Find why SQLFlow needs Workflow from [workflow](/doc/design/workflow.md); and
+1. How we implement it from [workflow package](/doc/design/workflow_package.md).
+
+The following figure shows a typical SQLFlow workflow architecture:
+
+![](figures/workflow.png)
+
+1. Users type in the SQL program on the client side, like Jupyter Notebook. the SQLFlow plugin on Notebook can send the SQL program to SQLFlow server via gRPC.
+1. SQLFlow gRPC server would access the gRPC request and translate it to Workflow YAML file, then submit to Kubernetes.
+1. Each of the circle from the above figure represents a workflow step, the workflow engine like Argo would execute them according to the dependency in YAML spec.
+
+## Prerequisites
```

Comment:
The following content are steps of the tutorial, but not a list of prerequisites. You might want to rename this section into something like `A Step-by-step Guide`.